controller/uninstall_controller.go

```diff
@@ -372,6 +378,13 @@ func (c *UninstallController) deleteCRDs() (bool, error) {
 		return true, c.deleteVolumes(volumes)
 	}
 
+	if snapshots, err := c.ds.ListSnapshots(labels.Everything()); err != nil {
+		return true, err
+	} else if len(snapshots) > 0 {
+		c.logger.Infof("Found %d snapshots remaining", len(snapshots))
+		return true, c.deleteSnapshots(snapshots)
+	}
```

Comment:
We deleted all volume CRs before deleting snapshot CRs in the above steps.

Since at this step the volume is already gone, we can delete any snapshot CRs in the system

Reply:
Got it. It's better to put this explanation as a comment. BTW, if there are still snapshot CRs here (after the volume cleanup), it implicitly means that these CRs are orphans, right?

Reply:
Yeah, those CR are orphans for a very short period of time. It will automatically be cleaned up by Kubernetes because the owner (the volume) is gone

Reply:
Update the comment. Thank you